far/changelog_eng

```diff
@@ -1,3 +1,7 @@
+MZK 11.07.2020 21:04:57 -0700
```

Comment:
Maybe it's time to merge changelogs together? Just asking.

Reply:
How exactly it would look like?

Reply:
Maybe something as simple as [far/changelog.md](https://github.com/MKadaner/FarManager/blob/mzk-changelog/far/changelog.md)? Formatting details may need to be worked out.

Reply:
Some points to consider:
- The files are huge: github already can't render changelog and changelog_eng is close to that. Merging them into even huger one won't improve the situation, so any formatting doesn't really make sense.
- It might be easier to update the merged version, but I'm not sure about reading it - a reader will have to jump over several lines over and over again.

Reply:
Let's move this conversation to an issue. I created #253.